test/unit/test_analysis_function.c

```diff
@@ -490,6 +495,11 @@ bool test_analysis_function_force_rename() {
 
 	// rename a to b, but b already exists, so we force rename
 	mu_assert_notnull(rz_analysis_function_force_rename(a, "b"), "function force rename");
+	// check whether the name begins with originally provided one
+	mu_assert_true(rz_str_startswith(a->name, "b"), "function renamed incorrectly");
```

Comment:
i'm not yet ok with this test.
I think it should be two functions defined `a` and `b` (at different addresses), then i would force rename `a` to be `b` and check that is `b_xxxx` (since you know the address, it should be easy to guess the name).

Reply:
also check the return value matches with what is in  `a->name`

Reply:
I first thought about doing it that way, but then it assumes many things about the internal implementation. I just want to assume the function gets renamed with a unique suffix. This allows the implementation to be changed in future without breaking the test

Reply:
no. the function is meant to return a certain result which we know being name_offset, we should test for this, not for anything else. also if in the future we change something we can always change the test.

Reply:
Ok, I'll make the change :-)